frontend-react/src/pages/resources/api-programmers-guide/documentation/Documentation.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import { MarkdownLayout } from "../../../../components/Content/MarkdownLayout";
+import Markdown from "../../../../content/resources/api-programmers-guide/documentation/Documentation.mdx";
+
+export interface DocumentationPageProps {}
+
+export function DocumentationPage() {
+    return (
+        <MarkdownLayout sidenav={<></>}>
```

Comment:
Can we just have a default within `<MarkdownLayout />` instead of having to pass an empty JSX fragment?

Reply:
This is intentional for the page as a sidenav will be given eventually. MarkdownLayout allowing for a full-width content section when not given a sidenav is the desired default behavior.